field-library/README.md

```diff
@@ -0,0 +1,15 @@
+<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+
+# Miranj Craft Boilerplate
+
+A bunch of ready to use fields provided by Miranj to speed-up your project development process. It uses the [Field Manager](https://github.com/verbb/field-manager) plugin to import the fields.
+
+## Usage
+
+1.  Open the Control Panel and go to _Field Manager_. ![Image for step 1](images/step_1.png 'Step 1')
+2.  Go to _import_ tab in _Field Manager_.
```

Comment:
1 & 2 can be combined: Go to _Field Manager_ > _Import_. No need for screenshot.